src/utils/set-deep.ts

```diff
@@ -66,7 +66,7 @@ export default function setDeep(
     if (!obj) {
       options.safeSet(target, prop, {});
     } else if (obj && target[prop] instanceof Change) {
-      if (typeof target[prop].value === 'object') {
+      if (isObject(target[prop].value)) {
```

Comment:
Looks like we might need an import?

Reply:
line 65? we got it alredy

Reply:
![image](https://user-images.githubusercontent.com/143068/82817590-895f4e00-9e9d-11ea-8134-4926a85a1aa5.png)

Reply:
oh. you mean we should use the one from is-object.ts?